.github/workflows/dockerimage.yml

```diff
@@ -22,11 +22,15 @@ jobs:
         login-server: healthplatformregistry.azurecr.io
         username: ${{ secrets.REGISTRY_USERNAME }}
         password: ${{ secrets.REGISTRY_PASSWORD }}
-    
+
+    - name: get-npm-version
+      id: package-version
+      uses: martinbeentjes/npm-get-version-action@master
```

Comment:
Instead of pulling this from the master branch, should we pull the specific `@v1.0.0` version? It won't make a difference right now, but might avoid issues in the future.

Reply:
Yes. As this pipeline runs only when master is updated,which means we are updating the newest code of the current version. We might need add branch trigger for old versions like v1.0.0/v2.0.0 respectively so that old images can get updated.

Reply:
I'm sorry, my comment wasn't specific. I meant pulling the 1.0.0 version of the npm-get-version-action instead of pulling its master version.
I agree with your statement though. Once we make the version specific branches we'll need to add them to this script.

Reply:
Got it. I agree, using a specific version totally make sense. Thanks!